website/content/posts/2023-02-02-one-year/_index.md

```diff
@@ -0,0 +1,132 @@
+  and their [interactions](https://github.com/PRQL/prql/issues/300),
+
+- fundamentals of how [functions](https://github.com/PRQL/prql/issues/444) and
+  pipelines are [evaluated](#define-functional-semantics).
+
+PRQL is now in a state where it can greatly improve the developer experience for
+writing complex analytical queries, but it does require a bit of fiddling to set
+up in your environment.
+
+In the coming year, we are aiming to improve that by providing a dbt plugin and
+integrations for tools like Rill-developer, Metabase, and DataGrip. Read more
+about our plans and ambitions in [the roadmap](https://prql-lang.org/roadmap/).
+
+## How are people using it?
+
+For data analytics at [SuperSimple](https://gosupersimple.com/):
+
+> We've been using PRQL under the hood to power complex analytics workflows at
+> Supersimple for more than 6 months now. The speed of iteration and response to
+> user feedback has been amazing during this period!
+>
+> PRQL is what I think SQL should have been like from day 1 and adopting it has
+> likely literally saved us months.
+
+For making
+[pretty reports with SQLite](https://prql-lang.org/posts/2023-01-28-format-pretty-reports/).
+
+For quick, readable
+[scripts in the command line](https://prql-lang.org/posts/2023-01-27-prql-query/).
+
+## What have you missed?
+
+In the past year, the community has created many things, some of which have not
+been noticed as much as they should have been. In the summary of the year, we
+want to fix that and put a spotlight on the amazing work that was done.
+
+### Playground
+
+We have a [Playground](https://prql-lang.org/playground/) that can compile and
+execute PRQL queries in-browser. It's using prql-compiler and DuckDB, both
+compiled to WASM modules.
+
+![PRQL Playground](URpCf29.png)
+
+Currently, this is the best way to see how the relation is manipulated instantly
+as you type the query.
+
+### VS Code extension
+
+Extension that provides syntax highlighting and compiled SQL within your editor.
+
+![VS Code extension](7cpDySb.png)
+
+### prql-query
+
+CLI tool that uses DuckDB and DataFusion to execute PRQL queries from your
+terminal. Useful for wrangling your CSV and parquet files on the go.
+
+![pq](ncVXken.png)
+
+### Define functional semantics
+
+Given the initial proposal of the language, we constructed consistent semantics
+of how functions in PRQL work, what they can express and how can they be abused.
+
+To keep this post brief, we'll expose a single snippet of what's possible and
+invite you to read more in
+[a recent post](https://prql-lang.org/functional-relations/) or in
+[the language documentation](https://prql-lang.org/book/internals/functional-lang.html).
+
+```prql
+func take_oldest n rel -> (
+    rel
+    sort [-invoice_date]
+    take n
+)
+
+from invoices
+take_oldest 3
+```
+
+> Function `take_oldest` is used in the main pipeline, just any other regular
+> transform.
+
+### Relational Query
```

Comment:
Great to include these two items!

Reply:
For each testimonial, you could include:

My sentiment after the Pretty Reports line: 

> I can no longer bring myself to write bare SQL - PRQL makes building queries so easy. 

And maybe this for the CLI...

> ... I can use `prql-query` to execute PRQL queries against database engines.

Reply:
It seems too promotional to quote ourselves.